brainstormforce/astra
# AST-3405 - SVG Icon Jerk

### Description

- After WordPress 6.3 release we changed editor eye icon dashicon to SVG, facing some jerk
- JIRA - https://brainstormforce.atlassian.net/browse/AST-3405

### Screenshots

https://d.pr/v/auRbyc

### Types of changes




Bug fix

### How has this been tested?

1. Create a new setup
2. Add new Post/Page
3. Set Network Throttling in Dev tools to "Fast 3G" you will see the Jerk.

### Checklist:
